bookserver/db.py

```diff
@@ -44,6 +44,7 @@
 else:
     connect_args = {}
 
+# TODO: databases doesn't yet support sqlalchemy 1.4, which has migration options to `sqlalchemy 2.0 mode <https://docs.sqlalchemy.org/en/14/changelog/migration_20.html#migration-to-2-0-step-four-use-the-future-flag-on-engine>`_. This is something we want to do from the beginning in our code!
```

Comment:
I know that the later version of 1.4 directly support async and await.  I'm unclear about the direction on fast api but it seems like using sqlalchemy directly with out databases in the middle would be optimal.

Reply:
I posted an [issue](https://github.com/bnmnetp/BookServer/issues/9) for this.